src/morale.h

```diff
@@ -22,6 +22,8 @@ enum morale_type : int
     MORALE_FEELING_GOOD,
     MORALE_SUPPORT,
     MORALE_PHOTOS,
+    MORALE_HAIRCUT,
+    MORALE_SHAVE,
```

Comment:
Let's just call the hilarious random morale effects that will result when someone loads an old save "works as intended". :P